src/main/java/gov/nih/nci/evs/api/service/AbstractStardogLoadServiceImpl.java

```diff
@@ -464,6 +476,22 @@ public void loadObjects(
     logger.info("Done loading Elastic Objects!");
   }
 
```

Comment:
Let's add documentation on what this class does (converts what to an AssociationEntry)